src/components/style/CustomBackdrop.tsx

```diff
@@ -0,0 +1,35 @@
+import { BottomSheetBackdropProps } from "@gorhom/bottom-sheet";
```

Comment:
this component could be in the ui folder

components/ui/CustomBackdrop

Reply:
Done: https://github.com/LangaraBeetles/weaup-react-native-app/pull/24/commits/3beaab4b7c02f88daf5b6efd1ef22b835100e1e4